bin/src/MixinGenerator.php

```diff
@@ -122,7 +122,7 @@ private function interface(ReflectionClass $assert): string
      */
     private function nullOr(ReflectionMethod $method, int $indent): ?string
     {
-        return $this->assertion($method, 'nullOr%s', 'null|%s', $indent);
```

Comment:
Why was this changed?

Reply:
The code generator now needs to generate `|null` to match php-cs-fixer expectations

Reply:
Because used for StyleCI base preset `symfony` includes rule `phpdoc_types_null_last` (https://docs.styleci.io/presets#symfony).

To follow this rule I needed to update this line (and below - line 265).
As an alternative - I can disable this rule if it is not expected to change order.